lte/cloud/go/services/subscriberdb/load.go

```diff
@@ -231,3 +231,37 @@ func ConvertSubEntsToProtos(ent configurator.NetworkEntity, apnConfigs map[strin
 
 	return subData, nil
 }
+
+func LoadSuciProtosPage(
+	ctx context.Context, networkID string) ([]*lte_protos.SuciProfile, error) {
+
+	network, _ := configurator.LoadNetwork(ctx, networkID, true, true, serdes.Network)
+
+	ngcModel := &lte_models.NetworkNgcConfigs{}
+	ret := ngcModel.GetFromNetwork(network)
+	if ret == nil {
+		return nil, nil
+	}
+
+	suciProfileList := ret.(*lte_models.NetworkNgcConfigs).SuciProfiles
+	suciProtos := []*lte_protos.SuciProfile{}
+	for _, suciProfile := range suciProfileList {
+		suciProto, _ := ConvertSuciEntsToProtos(suciProfile)
+
+		suciProtos = append(suciProtos, suciProto)
+	}
+
+	return suciProtos, nil
+}
+
+func ConvertSuciEntsToProtos(ent *lte_models.SuciProfile) (*lte_protos.SuciProfile, error) {
```

Comment:
Move this to the `lte_models` package and make it a method instead of a func

Reply:
Why does this func return an err?

Reply:
resolved.